devel/ci/bodhi-ci

```diff
@@ -751,6 +769,72 @@ class UnitJob(Job):
         return self
 
 
+class IntegrationBuildJob(BuildJob):
+    """
+    Define a Job for building container images for integration testing.
+
+    See the Job superclass's docblock for details about its attributes.
+    """
+
+    _label = 'integration-build'
+
+    def __init__(self, *args, **kwargs):
+        """
+        Initialize the IntegrationBuildJob.
+
+        See the superclass's docblock for details about accepted parameters.
+        """
+        super(IntegrationBuildJob, self).__init__(*args, **kwargs)
+
+        self._container_image = '{}-integration/{}'.format(CONTAINER_NAME, self.release)
+        dockerfile = os.path.join(PROJECT_PATH, 'devel', 'ci', 'integration', 'bodhi', 'Dockerfile')
+        self._command = [container_runtime, 'build', '--pull', '-t', self._container_image,
+                         '-f', dockerfile, '.']
+
+
+class IntegrationJob(Job):
+    """
+    Define a Job for running the integration tests.
+
+    See the Job superclass's docblock for details about its attributes.
+
+    Attributes:
+        pyver (int): The Python version we are testing.
+    """
+
+    _label = 'integration'
+
+    def __init__(self, archive, archive_path, pyver, *args, **kwargs):
+        """
+        Initialize the IntegrationJob.
+
+        See the superclass's docblock for details about additional accepted parameters.
+
+        Args:
+            archive (bool): If True, set up the volume mount so we can retrieve the test results
+                from the container.
+            archive_path (str): A path on the host to share as a volume into the container for
+                its /results path.
+            pyver (int): Which major version of Python we are testing with.
+        """
+        super(IntegrationJob, self).__init__(*args, **kwargs)
+
+        self.pyver = pyver
+        self._label = '{}-py{}'.format(self._label, pyver)
+        self._command = [sys.executable, '-m', 'pytest', '--no-cov', 'devel/ci/integration/tests/']
```

Comment:
Won't this be the Python 3 since this script is executed with Python 3? We probably want to use ```pyver``` here.

Reply:
Actually the python bin can come from a virtualenv, so just using `python` or `python-pyver` isn't sufficient. Is there a problem with `sys.executable`?

Reply:
Won't ```sys.executable``` reference the Python executable that ```bodhi-ci``` was run with? This means that the ```-p``` flag won't work, right?

Reply:
Ah, this would be the Python that executes the test suite itself, not the version that executes Bodhi. I think my mind was just in a weird place. I think this is OK.